05_silly_blocks/silly_blocks.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+def reverser
+  str = []
+  rev = yield.split(' ')
+  rev.each do |word|
+    temp = word.reverse
+    str.push(temp)
+  end
+  str.join(' ')
+end
+
+def adder(num = 1, &block)
+  block.call + num
+end
+
+def repeater(num = 1, &block)
+  for i in (1..num) do
```

Comment:
NO NO NO, forget about `for` loops! Ruby gives you `times` and `each`, why would you touch this ugly shit?)